src/core-services/product/schemas/product.graphql

```diff
@@ -30,6 +30,9 @@ type Product {
   "Subtitle"
   pageTitle: String
 
+  "Price range"
+  price: ProductPriceRange
+
```

Comment:
This stuff should be added by extending in `src/plugins/simple-pricing/schemas/schema.graphql` rather than here. Same way it's done for catalog.

Reply:
Is the standard `price` object even still used on products or is it now `pricing`?

Reply:
Moved to `simple-pricing` schema